pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+		xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>BBFW</groupId>
+	<artifactId>traffic_simulation</artifactId>
+	<version>1.0-SNAPSHOT</version>
```

Comment:
Ne, das `-SNAPSHOT` gehört da hin, denn wie du zitierst markiert das Versionen, die nicht released sind. Das gilt für dieses Projekt noch eine Weile - man macht man wirklich erst weg, wenn das Release gemacht und direkt danach auch wieder hin (dann z.B. mit `2.0-SNAPSHOT`).

Der Grund dafür ist, dass Maven nicht zulässt, Versionen zu aktualisieren. Du kannst nicht erst eine Version 1.0 installieren (mit `mvn install`) und nachher eine andere (außer du löschst manuell Dateien). Da man in der Entwicklung aber dauernd `install` aufruft, würde das zu Problemen führen. Das `-SNAPSHOT` am Ende hat diese Beschränkung nicht.